assistant_dists/dream_kg/dev.yml

```diff
@@ -91,4 +91,16 @@ services:
       - "~/.deeppavlov:/root/.deeppavlov"
     ports:
       - 8027:8027
+  dff-user-kg-skill:
+    volumes:
+      - "./skills/dff_user_kg_skill:/src"
+      - "./common:/src/common"
+    ports:
+      - 8132:8132
+  dff-travel-italy-skill:
+    volumes:
+      - "./skills/dff_travel_italy_skill:/src"
+      - "./common:/src/common"
+    ports:
+      - 8139:8139
```

Comment:
please add this skills to components.tsv (as I do not see this file to be modified in this PR)